### PR TITLE
fix: update support resources in dashboard for institutional managers

### LINKED ISSUE #39: Build institutional manager dashboard(s)
As an institutional manager, I should see a dashboard with information specific to my institution after logging into to Pressbooks. See https://www.figma.com/file/hAvkPJpDgJDDyh8g358AqR/PB-Site-Adjustments?type=design&node-id=32-9467&mode=design&t=MCrLdVEuEnZGwGWb-0.

Screenshot from 2024-02-14 07-04-37

The topmost and bottom most blocks in the dashboard should be identical to those currently displayed to super admins/network managers. The middle sections are different. The left most block should have the heading 'Institutional Usage' and should display the institution's name and the number of books & users assigned to that institution. The 'Explore stats' button should link to the stats page _for that institution_. The 'Administer Institution' section should include links to the book and user list pages _for that institution_. 

DoD:
1. When an institutional manager logs in to a Pressbooks network, they're automatically routed to the correct institutional dashboard for their user
2. The institutional dashboard includes correct links (or link placeholders) for the stats page, book list, and user list _for their institution_
3. Institutional managers cannot load the network dashboard or an institutional dashboard for other institutions by accident

## Notes
My idea was that every institution would have their own dashboard page (each one would have an institution ID parameter in the URL). Institutional managers would get routed to the relevant institution dashboard when logging in. Super admins and network managers could view these pages by entering the URL directly and would see the same thing that the institutional managers sees.

### PR DESCRIPTION
**Pull request by ho-man-chan**
Resolves: https://github.com/pressbooks/pressbooks-multi-institution/issues/39

Test: As a network manager, navigating to the dashboard should see the correct support resources.
image

### CODE REVIEW

File: resources/views/dashboard/institutional.blade.php [hunk 1]
```diff
@@ -64,47 +64,46 @@ class="pb-dashboard-flex-image"
...
 							<img
-								src="{{ PB_PLUGIN_URL . "assets/dist/images/pb-guide.png" }}"
+								src="{{ PB_PLUGIN_URL . "assets/dist/images/pb-network-guide.png" }}"
 								alt=""
 							/>
-							{{ __('Pressbooks user guide', 'pressbooks-multi-institution' )}}
+							{{ __('Network manager guide', 'pressbooks-multi-institution' )}}
 						</a>
-						<p>{{ __( 'Find help and how-tos for your publishing project in this detailed handbook.', 'pressbooks-multi-institution' ) }}</p>
+						<p>{{ __( 'Learn how to administrator your Pressbooks network from our comprehensive how-to guide.', 'pressbooks-multi-institution' ) }}</p>
```
Comment (SteelWagstaff) [inline resources/views/dashboard/institutional.blade.php]:
```suggestion
						<p>{{ __( 'Learn how to administer your Pressbooks network from our comprehensive how-to guide.', 'pressbooks-multi-institution' ) }}</p>
```

File: resources/views/dashboard/institutional.blade.php [hunk 2]
```diff
@@ -64,47 +64,46 @@ class="pb-dashboard-flex-image"
...
-					</li>
 					<li class="resources" id="pressbooks-guide">
 						<a href="https://guide.pressbooks.com" target="_blank">
 							<img
-								src="{{ PB_PLUGIN_URL . "assets/dist/images/pb-guide.png" }}"
+								src="{{ PB_PLUGIN_URL . "assets/dist/images/pb-network-guide.png" }}"
 								alt=""
 							/>
-							{{ __('Pressbooks user guide', 'pressbooks-multi-institution' )}}
+							{{ __('Network manager guide', 'pressbooks-multi-institution' )}}
```
Comment (SteelWagstaff) [inline resources/views/dashboard/institutional.blade.php]:
```suggestion
							{{ __('Network manager guide', 'pressbooks-multi-institution' ) }}
```

**Comment by SteelWagstaff** [comment]
Maybe this is a dumb question, but couldn't this whole section simply be loaded from pressbooks/pressbooks: https://github.com/pressbooks/pressbooks/blob/5251bdb47a173aba0fb6d0724a9707e42ad6b037/templates/admin/dashboard/network.blade.php#L171-L221? I thought the point of Blade templates was avoiding repeating code in multiple locations

**Comment by ho-man-chan** [comment]
> Maybe this is a dumb question, but couldn't this whole section simply be loaded from pressbooks/pressbooks: https://github.com/pressbooks/pressbooks/blob/5251bdb47a173aba0fb6d0724a9707e42ad6b037/templates/admin/dashboard/network.blade.php#L171-L221? I thought the point of Blade templates was avoiding repeating code in multiple locations

We probably could if we rearrange some things. What do you think @arzola? Is it worth it?

**Comment by arzola** [comment]
I think is doable, but that would require to move that part into a isolated blade file and load it from there, we have the pressbooks templates namespace already loaded, so it would be a matter of render that partial inside of multi-institution

**Comment by ho-man-chan** [comment]
Update:
- made a separate blade file for the support resource block in pressbooks/pressbooks and this plugin will include it rather than duplicating code. thanks for the suggestions and help from steel and oscar.

**Pull request closed without merging**